docs/advanced/solo-proof-of-stake-voting.md

```diff
@@ -0,0 +1,198 @@
+
+---
+
+# Setting Up A Secure "Cool" Wallet
+## Hardware
+The first concept we will address is that of _compartmentalization_. There is really no reason that a machine which may be storing large amounts of cryptocurrency should have multiple roles. It should not be used, for gaming, word processing, web browsing, downloading torrents, etc...
+
+I highly recommend [Qubes OS](https://www.qubes-os.org/) paired with [compatible hardware](https://www.qubes-os.org/hcl/).
+For reference it runs really well on a Thinkpad X260 and even an older Thinkpad X220.
+
+If cost is a huge factor, single-board computers such as the Raspberry Pi 3 Model B can also be tasked for this purpose however using a machine that is more powerful and can handle Qubes is preferable.
+
+---
+
+## Configuration
+Once your staking machine is set up you will need to install the Decred CLI tools.
+1. Import the Decred Release Signing Key in GnuPG.
+
+`gpg --keyserver pgp.mit.edu --recv-keys 0x518A031D`
+
+2. Download the installer, manifest, and signature files.
+
+`wget https://github.com/decred/decred-release/releases/download/v1.3.0/{dcrinstall-linux-amd64-v1.3.0,manifest-dcrinstall-v1.3.0.txt,manifest-dcrinstall-v1.3.0.txt.asc}`
+
+3. Verify the manifest.
+
+`gpg --verify manifest-dcrinstall-v1.3.0.txt.asc`
+
+4. Verify the SHA-256 hash in the manifest matches that of the binary.
+
+`sha256sum dcrinstall-linux-amd64-v1.3.0`
+
+5. Make the binary executable.
+
+`chmod +x dcrinstall-linux-amd64-v1.3.0`
+
+6. Run it.
+
+`./dcrinstall-linux-amd64-v1.3.0`
+
+**Important**
+
+When you create your wallet, write the 33 seed words down using a pen and paper on a hard surface, and write out two copies.
+
+* **DO NOT** take a picture of the seed words with your phone’s camera.
+* **DO NOT** save the seed words in a text file on a computer.
+* **DO NOT** email the seed words to yourself for "safe keeping".
+
+The only way a person should be able to get the seed words is by acquiring one of the copies you have written out. You should store one copy offsite in case of fire or theft.
+
+---
+
+Once the wallet is created we can do a few things to make life easier. 
+
+1. Create a `bash` script called `decred.sh` which will start a `tmux` session for each application, start `dcrd`, start `dcrwallet`, prompt us for the password to unlock the wallet, and start `dcrctl`. Also make sure you have `tmux` installed.
+
+`echo "tmux new -d -s dcrd 'dcrd & tmux new -d -s dcrwallet 'dcrwallet --promptpass' & tmux attach -t dcrwallet" > ~/decred.sh"`
+
+2. Add the pathe to the Decred binaries to your `.profile`.
+`"PATH=~/decred:$PATH" >> ~/.profile && source ~/.profile`
+
+3. Now make it executable with `chmod +x ~/decred.sh`
+
+Now you can start `dcrd`, `dcrwallet`, and unlock it using `promptsecret` just by running `./decred.sh`.
```

Comment:
Actually it looks like the script is not using `promptsecret`, but it is using the `--promptpass` flag instead